css/main.css

```diff
@@ -326,68 +521,112 @@ header .menu img {
   font-weight: bold;
 }
 
-.skills .skill .skill_li {
+.skill__li {
   color: var(--white);
   font-size: 2.2rem;
 }
 
-.form {
+@media (min-width: 768px) {
+  .skills {
+    max-width: 115rem;
+    margin: 14rem auto;
+    display: flex;
+    justify-content: space-between;
+  }
+}
+
+/* ----------Form ------- */
+
+.form__content {
   padding: 2.4rem;
 }
 
-.form form {
+.form {
   display: flex;
   flex-direction: column;
   align-items: center;
   margin-bottom: 10rem;
 }
 
-.form .form_p {
-  font-size: 32px;
-  margin-top: 100px;
+.form_p {
+  font-size: 3.2rem;
+  margin-top: 10rem;
   color: #172b4d;
   font-weight: bold;
-  margin-bottom: 60px;
+  margin-bottom: 6rem;
   text-align: center;
 }
 
 .form input {
   width: 100%;
-  padding: 20px;
-  margin-bottom: 25px;
+  padding: 2rem;
+  margin-bottom: 2.5rem;
   border: none;
   border-bottom: 1px solid #d8d8d7;
 }
```

Comment:
- [ ] Kindly add a `font-family` to all input fields to make it match Figma ✨ 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/186997355-32845d1f-7e12-46e4-aa2d-7c1c3fb22117.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/186997454-3589bd0d-10af-49c7-ae5c-431892e7ce4a.png)